### PR TITLE
zdk and hooks update

### PR DESCRIPTION
**Pull request by anader123**
-Added docs and guide for the new ZDK that supports the Zora API 
-Updated NFT Hooks docs

### CODE REVIEW

**Review by erikreppel — APPROVED** [review]

File: docs/developer-tools/nft-hooks/usenftmetadata.mdx [hunk 1]
```diff
@@ -14,21 +14,37 @@ Most metadata servers allow for remote JSON fetches, however, there is a chance
...
+/**
+ * @param uri URI of metadata to fetch
+ * @returns @type useNFTMetadataType
+ */
+export function useNFTMetadata(
+  uri?: string,
+)
+```
+
+### Reponse Shape
```
Comment (IsabellaSmallcombe) [inline docs/developer-tools/nft-hooks/usenftmetadata.mdx:37]: spelling

File: docs/zora-api/intro.mdx [hunk 2]
```diff
@@ -21,7 +21,9 @@ The ZORA API makes it easy to retrieve NFT metadata, monitor on-chain events, an
 
 API Endpoint: https://api.zora.co/graphql
 
-<!-- [SDK](./zdk-api): Use the API without needing to know GraphQL -->
+[SDK](./zdk): A development kit to use the API without needing to know GraphQL
+
+[NFT Hooks](../developer-tools/nft-hooks/intro): A tool that makes NFT data retriveal even easier than the ZDK
```
Comment (IsabellaSmallcombe) [inline docs/zora-api/intro.mdx:26]: retrieval spelling

File: docs/zora-api/zdk.mdx [hunk 3]
```diff
@@ -0,0 +1,582 @@
...
+| timeFilter | A filter for getting sales during a range of time |
+| timeFilter.endDate | A string array of addresses that have bid for this NFT |
+| timeFilter.lookbackHours | An enum specifying the version of Zora the order is made on | V1Ask, V1BidShare, V1Offer, V2Auction, V3Ask
+| timeFilter.startDate | An enum specifying status of orders being returned | Active, Canceled, Completed
+
+---
+
+### search
+
+Searchs for an NFT or collection based on a string input.
```
Comment (IsabellaSmallcombe) [inline docs/zora-api/zdk.mdx:497]: Search? i don't think searchs is a word

File: src/components/content/ToolsCardsContent.js [hunk 4]
```diff
@@ -12,26 +12,26 @@ export const toolsCardsContent = [
...
     toolsParagraph:
-      'Tools to retrieve pricing and NFT information to display on your own website.',
-    toolsLink: '/docs/developer-tools/nft-rendering/introduction',
+      'A developer kit that makes it easy to use the Zora API without needing to know GraphQL.',
+    toolsLink: '/docs/zora-api/zdk',
   },
   {
     toolsTitle: 'Editions',
-    toolsParagraph: 'A creator toolkit makes it easy to create an NFT collection.',
+    toolsParagraph: 'An interface that makes it easy to create an NFT collection with no coding requried.',
```
Comment (IsabellaSmallcombe) [inline src/components/content/ToolsCardsContent.js:28]: required is spelled wrong

**Pull request merged**